installer/scripts/service_control

```diff
@@ -283,12 +471,16 @@ enable_omsagent_service()
             elif [ -x /sbin/chkconfig ]; then
                 /sbin/chkconfig --add $OMSAGENT_WS > /dev/null
             else
-                echo "Could not find a service controller to configure the OMS Agent Service."
-                exit 1
+                echo "FATAL:  Could not find a service controller to configure the OMS Agent Service."
+                exit 9
```

Comment:
Make this code constant
Same multiple workspaces concern: this method should return, not exit

Reply:
Same response.

Reply:
Let's discuss this tomorrow